ansible/configs/sap-smart/files/cloud_providers/osp_cloud_template_master.j2

```diff
@@ -0,0 +1,232 @@
+#jinja2: lstrip_blocks: "True"
+---
+heat_template_version: 2018-03-02
+
+description: >-
+  Top level HOT for creating new project, network resources and instances.
+  This template relies on ResourceGroups and a nested template that is
+  called to provision instances, ports, & floating IPs.
+
+resources:
+
+  {{ guid }}-infra_key:
+    type: OS::Nova::KeyPair
+    properties:
+      name: {{ guid }}-infra_key
+      save_private_key: true
+
+{% if osp_project_create | bool %}
```

Comment:
This should be removed as well as the `project_role_user` section. These are handled in an earlier playbook now.

Reply:
Done